dp/cloud/python/magma/radio_controller/tests/test_utils/active_mode_cbsd_builder.py

```diff
@@ -88,6 +90,11 @@ def with_state(self, state: CbsdState) -> ActiveModeCbsdBuilder:
         self.state = state
         return self
 
+    def with_grant_settings(self, grant_redundancy_enabled: bool, max_ibw_mhz: int):
```

Comment:
Can we also add the other fields (or some default values for them), so that we test all of the fields that we are using?